### DISCUSSION TITLE
DAI 2024-2025 - Practical work 3 - Gianni Cecchetto, Nathan Tschantz and Zaïd Schouwey

### DISCUSSION THREAD

**Discussion by GianniCecchetto** [heig-vd-dai-course/heig-vd-dai-course#531] [Show and tell]
For this practical work, we would like to do it with the project we have in BDR with Nathan. Our group in BDR is named embedded.

**Comment by GeraudSilvestri** [comment]
We need more details of what your app will do

**Reply by GianniCecchetto** [reply]
Our app will be able to create and manage musical events. We will be able to create, modify and delete Artists, Spectators, Contributors, Events, Concerts, Group of artists, Small restauration zone in the scope of an event.

**Reply by ludelafo** [reply]
Good for me, thanks!

**Comment by GianniCecchetto** [comment]
Rendu :
Lien sur le repo : [github.com/GianniCecchetto/dai-practical-work-3](https://github.com/GianniCecchetto/dai-practical-work-3)
Hash dernier commit : 2f496ff358f98292db07e1086120611060a4e7e1
